packages/create-app/templates/template-tutorial-todo-aip-app/src/Home.tsx

```diff
@@ -0,0 +1,93 @@
+import aipLogo from "/aip-icon.svg";
+import { useCallback, useEffect, useState } from "react";
+import CreateProjectButton from "./CreateProjectButton";
+import CreateTaskButton from "./CreateTaskButton";
+import DeleteProjectButton from "./DeleteProjectButton";
+import css from "./Home.module.css";
+import Layout from "./Layout";
+import type { MockProject } from "./mocks";
+import ProjectSelect from "./ProjectSelect";
+import TaskList from "./TaskList";
+import useProjects from "./useProjects";
+
+function Home() {
+  const [projectId, setProjectId] = useState<string | undefined>(undefined);
+  const { projects, updateProjectDescription } = useProjects();
+  const project = projects?.find((p) => p.id === projectId);
+  const [isProcessing, setIsProcessing] = useState<boolean>(false);
+
+  const handleSelectProject = useCallback(
+    (p: MockProject) => setProjectId(p.id),
+    [],
+  );
+
+  const handleProjectDescriptionRecommendation = useCallback(async () => {
+    if (project == null) {
+      return;
+    }
+    setIsProcessing(true);
+    await updateProjectDescription(project);
+    setIsProcessing(false);
+  }, [project, updateProjectDescription]);
+
+  useEffect(() => {
+    if (project == null && projects != null && projects.length > 0) {
+      setProjectId(projects[0].id);
+    }
+  }, [project, projects]);
+
+  return (
+    <Layout>
+      <div className={css.tutorialBannerWrapper}>
+        <div className={css.tutorialBanner}>
+          <p className={css.tutorialBannerTitle}>
+            💡 Welcome to the To-do AIP tutorial!
```

Comment:
Also, I think we need to consistently either do Todo or To Do for example in Layout.tsx
```suggestion
            💡 Welcome to the To Do AIP tutorial!
```